README.md

```diff
@@ -1,51 +1,258 @@
-# Capitalisk Core
-Standalone Capitalisk node. Built using [LDEM](https://github.com/jondubois/ldem).
+# Capitalisk Node
+
+## 1. Install essentials
+
+```sh
+sudo apt-get install build-essential python
```

Comment:
Can we mention that this is optional (e.g. if the user wants optional support for SQLite)?
We should always aim to minimize the number of dependencies.

Reply:
Ohh makes sense, I wasn't sure why this was mentioned though

Reply:
I believe we should remove it, since we already have binaries hosted, and it's not just this dependency they need to install when SQLite can't fetch right binaries